### DISCUSSION TITLE
baker数据集从头训练无法生成音频

### DISCUSSION THREAD

**Discussion by aluminumbox** [fishaudio/Bert-VITS2#316] [Q&A] [answered]
使用训练数据集：baker，1万条训练数据
训练完毕后，发现生成的音频基本听不出内容。如果使用下载的base model去预训练后，是可以生成正常音频的。
请问这个可能是怎么回事？baker数据集在wetts的vits框架下也试过了是可以正常生成音频的。
请有经验的大佬指点一下，谢谢！

**Comment by Stardust-minus** [comment, marked as answer]
不推荐从头。需要很多trick和GPU（

**Reply by aluminumbox** [reply]
谢谢指点，明白了
